src/boot.tsx

```diff
@@ -22,6 +22,17 @@ export default (element: HTMLDivElement, hub: Hub) => {
     applyMiddleware(createEpicMiddleware(rootEpic))
   );
 
+  const getPackages = (packages: string) => {
+    if (packages){
+      return `from dcl_package_manager import install_packages; install_packages([${formatPackages(packages)}])\n`;
+    }
+    return "";
+  };
+
+  const formatPackages = (packages: string) => {
+    return packages.split(",").reduce((accumulator, single_package) => accumulator + `'${single_package}',` , "");
```

Comment:
I would add the accumulator inside the templates string. Also, can you rename `single_package` to `singlePackage`.